src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/tabType';
+
+type Props = {
+  selectedTabId: string,
+  tabs: Tab[],
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  selectedTabId,
+  tabs,
+  onTabSelected,
+}) => {
+  // eslint-disable-next-line max-len
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab: Tab) => (
```

Comment:
You don't really need to add a type for a separate tab, because you already have a type for tabs.
```suggestion
          {tabs.map(tab => (
```